cartoframes/data/services/bq_datasets.py

```diff
@@ -162,7 +185,45 @@ def status(self):
     def result(self):
         status = self.status()
 
-        while status != 'done' and status != 'failed':
+        while status not in ('success', 'failure'):
+            status = self.status()
+
+        return status
+
+
+class BQUserEnrichmentJob:
+
+    def __init__(self, job_id):
+        self.id = job_id
+        # TODO fix this crap
+        self.session = requests.Session()
+        self.api_key = 'my_valid_api_key'
+
+    def status(self):
+        url = '{}/enrichment/{}/status'.format(DO_ENRICHMENT_API_URL, self.id)
```

Comment:
I changed the URL to `/bq/enrichment`

Reply:
`DO_ENRICHMENT_API_URL = 'http://localhost:7070/bq'`